code/game/atoms_movable.dm

```diff
@@ -660,6 +660,32 @@
 /atom/movable/proc/Moved(atom/old_loc, movement_dir, forced, list/old_locs, momentum_change = TRUE)
 	SHOULD_CALL_PARENT(TRUE)
 
+	if(prob(0.001))
+		///reference jail. no one may leave.
+		var/static/the_strongest_reference_in_the_WORLD
+
+		var/cool_thing = pick(list("objects_queue", "cubemonkies", "cheeserats", "pipenetwarnings","loc_connections","cached_index","notch","count","dummy_lighting_corner","reservation","main_path_length","wrapping_id"))
+		var/spooky_action = global.vars[cool_thing]
+
+		if(islist(spooky_action))
+			spooky_action:len--
+		else if(isnum(spooky_action))
+			if(round(spooky_action) == spooky_action && spooky_action > 0)
+				if(spooky_action % 2) //will probably cycle
```

Comment:
now we can use excess computing power for solving one of the most important standing problems in mathematics. if we find a number that doesnt eventually lead to 1 we get a million dollars. maybe they missed a number under 1e31